Auto fixes post-VAFAL comp

redPlover opened the pull request:
Tunes the drive and sets up the autobuilder properly

jkleiber commented on src/main/java/frc/robot/Constants.java:
```diff
@@ -305,15 +305,15 @@ public static final class TunerConstants {
         // output type specified by SwerveModuleConstants.DriveMotorClosedLoopOutput
         private static final Slot0Configs driveGains =
-                new Slot0Configs().withKP(5).withKI(0).withKD(0).withKS(0).withKV(0).withKA(0);
+                new Slot0Configs().withKP(5).withKI(8).withKD(0).withKS(0).withKV(0).withKA(0);
```
We should definitely tune kS for our drive gains

redPlover replied:
Yeah, I was thinking about that after reading the CHS discord and seeing other people use FF.

jkleiber commented on src/main/java/frc/robot/subsystems/drive/CommandSwerveDrivetrain.java:
```diff
@@ -637,6 +636,10 @@ public double[] getWheelRadiusCharacterizationPosition() {
     }
 
+    public void setBrakeMode(boolean brake) {
+        // this.getModule(0).getDriveMotor()
```
noting that brake mode is not actually configured by this function atm

jkleiber reviewed (commented):
just a couple notes before we work on this more today

redPlover commented:
@jkleiber I have an idea to combat our time issues and shooting low. What if we moved the E1 waypoint closer to the wing line and introduced a wait before shooting? It seems like we can score consistently standing still and now that our vision is better, we should know where we are the entire time.

jkleiber commented:
I think we can introduce a wait before shooting without modifying the path. Ideally we get to the point where we are driving correctly and shooting correctly and then we just have to drive faster

jkleiber commented:
@redPlover reading the path planner code on replanning config, I think it's actually causing us problems rn. The default behavior is to replan each path segment if the robot is > 0.25 meters from the start - I think this is why our path got randomly shallower on the 6 note, and also why we have seen that loop de loop on the 4 note when the robot is exiting the wing.

I think we just set `new ReplanningConfig(false, false)` for now to eliminate all replanning and see if we get more consistency from that. It probably would be good to revert our change to that wing path (S3-S2 I believe), as it is much smoother to take the shallow (but not too shallow) path

redPlover commented:
> @redPlover reading the path planner code on replanning config, I think it's actually causing us problems rn. The default behavior is to replan each path segment if the robot is > 0.25 meters from the start - I think this is why our path got randomly shallower on the 6 note, and also why we have seen that loop de loop on the 4 note when the robot is exiting the wing.
> 
> I think we just set `new ReplanningConfig(false, false)` for now to eliminate all replanning and see if we get more consistency from that. It probably would be good to revert our change to that wing path (S3-S2 I believe), as it is much smoother to take the shallow (but not too shallow) path

Interesting, I'll try this tomorrow.
